### PR TITLE
[ONNX] Support large attribute and subgraph for large model

### PR DESCRIPTION
**Pull request by BowenBao**
Previously large tensor data in attributes and subgraphs are not stored externally. ONNX won't be able to serialize the model for cases where the total size sums up to >= 2GB. This PR enables that.

### CODE REVIEW

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 1]
```diff
@@ -197,6 +197,27 @@ def test_mobilenet_v2_with_external_data(self):
         self.run_model_test_with_external_data(model, x, rtol=1e-3, atol=1e-5,
                                                ort_optim_on=False)
 
+    @skipIfUnsupportedMinOpsetVersion(9)  # Because external data format was released with Opset 9.
+    def test_attribute_with_external_data(self):
+        class LargeModel(torch.nn.Module):
+            def forward(self, x):
+                return x + torch.ones(2, 1024)
```
Comment (neginraoof) [inline test/onnx/test_pytorch_onnx_onnxruntime.py:204]: Which attribute is ONNX node is this using for export?

**Review by neginraoof — APPROVED** [review]
LGTM, thanks!
About tests: since these tests won't fail even without your changes, is there a way to check external data export or files, maybe in test_utility?

File: test/onnx/test_pytorch_onnx_onnxruntime.py (hunk 1 above)
Comment (BowenBao) [reply test/onnx/test_pytorch_onnx_onnxruntime.py:204]: attribute value_t of Constant node.

**Comment by BowenBao** [comment]
@houseroad please take a look, thanks!

**Review by houseroad — APPROVED** [review]
Looks good.

**Pull request closed without merging**